spec/helpers.lua

```diff
@@ -237,6 +240,14 @@ _G.kong.core_cache = {
   end
 }
 
+-- restore global router_flavor
+if router_flavor then
+  _G.kong.configuration = {
+    router_flavor = router_flavor,
+    remove_sensitive = _G.kong.configuration.remove_sensitive,
+    }
+end
```

Comment:
It is a workaround to let the ci tests green.

We can set the global `kong.configuration.router_flavor`, but the `conf_loader()`
will load the test conf and set it to the default value `traditioinal_compatible`.

To let the entities(routes) get the correct flavor as expect,
I have to replace the table `kong.configuration`.

I think it is not a good solution, could someone show me a better way?